receiver.html

```diff
@@ -47,6 +50,10 @@
                     }
                 )
         };
+        
+        // Your codes
```

Comment:
This is not the correct way to assign pcode and playerID params. These 2 must be inside var pageLevelParams = {}.
OO.__internal is internal use only, and not for customer use case.